mlrun/projects/project.py

```diff
@@ -2012,7 +2012,7 @@ def enable_model_monitoring(
         base_period: int = 10,
         image: str = "mlrun/mlrun",
         deploy_histogram_data_drift_app: bool = True,
```

Comment:
I remember we agreed on adding an waiting flag with default set to false.
You can use `_wait_for_function_deployment()`.

Reply:
It's out of scope for the current PR, but I agree it can be helpful. Let's address it in another PR.